.github/workflows/deploy-operational-updates.yml

```diff
@@ -46,8 +46,12 @@ jobs:
         uses: aws-actions/amazon-ecr-login@v2
         id: login-ecr
 
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v3
+      - name: Set up Depot CLI (for Docker builds)
+        uses: depot/setup-action@v1
+
+      - name: Configure Docker with Depot
+        run: |
+          depot configure-docker --project ${{ secrets.DEPOT_PROJECT }} --token ${{ secrets.DEPOT_TOKEN }}
```

Comment:
This use Depot when invoking Docker with `docker build` or `docker buildx build`. The secrets have already been registered. So the existing deploy command should automatically use Depot.